ai-service/tests/test_llm_and_search.py

```
@@ -1,14 +1,14 @@
 from ai_service import db, embedder, ollama_client
+import pytest
 
 
 # -------------- LLM Chat Tests --------------
 
 
-def test_llm_chat_basic_response(monkeypatch):
-    # Mock Ollama response for deterministic test
+def test_llm_chat_basic_response(monkeypatch: pytest.MonkeyPatch):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_llm_chat_basic_response`

Add return type annotation: `None`

(ANN201)